### PR TITLE
align core service logs

### LINKED ISSUE #803: Align Core service logs.
**Description**
Here is an example of the logs generated by running a core service, executing two tasks and stopping the service. 
```
[2024-02-07 10:27:44][Taipy][INFO] Start job dispatcher...
[2024-02-07 10:27:48][Taipy][INFO] job JOB_arima_training_7d61b95e-4998-4696-953b-ff74179f06b3 is completed.
[2024-02-07 10:27:49][Taipy][INFO] job JOB_arima_scoring_9c374054-cd38-4abf-82bd-28f43c33de56 is completed.
[2024-02-07 10:27:49][Taipy][INFO] Core service has been stopped.
```
Running the core service actually adds a log about starting a job dispatcher. 
1. This is inconsistent regarding the log generated when stopping the service, 
2. Running the core service does more than just starting a job dispatcher

Any change on that should be reflected in the documentation (user manual, reference manual, demos, tips and tricks, etc...)

**Solution proposed**
Remove the log "Start job dispatcher..." and add a new log "Core service started..."

**Acceptance Criteria**
- [ ] The log should be consistent and representative of what is done.
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Ensure the documentation sets are updated

### PR DESCRIPTION
**Pull request by ooooo-create**
Resolves #803

### CODE REVIEW

File: taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def stop(self, wait: bool = True, timeout: Optional[float] = None):
             self.join(timeout=timeout)
 
     def run(self):
-        self._logger.info("Start job dispatcher...")
+        self._logger.info("Core service started...")
```
Comment (trgiangdo) [inline taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py]:
```suggestion
        self._logger.debug("Start job dispatcher...")
```

**Review by trgiangdo — CHANGES_REQUESTED** [review]
Thank you @ooooo-create for the fix.

However, the log "Core service started..." is misleading from the `_job_dispatcher.py`.

Instead, I suggest changing the "Start job dispatcher" log to a debug log, and add the "Core service started..." log to the `Core.run()` method.

Let me know what you think.

**Comment by ooooo-create** [comment]
Thanks!
> However, the log "Core service started..." is misleading from the `_job_dispatcher.py`.

yes, as issue #803 say that running the core service does more than just starting a job dispatcher, so just replace the words is misleading 

> Instead, I suggest changing the "Start job dispatcher" log to a debug log, and add the "Core service started..." log to the `Core.run()` method.

Also the `Core service has been stopped.` is add to the `Core.stop()` method, i think add  "Core service started..." log to the `Core.run()` method is a good way.

If set `Start job dispatcher` to a debug log , whether the `stop job dispatcher` also do that

File: taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def stop(self, wait: bool = True, timeout: Optional[float] = None):
             self.join(timeout=timeout)
 
     def run(self):
-        self._logger.info("Start job dispatcher...")
+        self._logger.debug("Start job dispatcher...")
```
Comment (trgiangdo) [inline taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py]: What do you think?
```suggestion
        self._logger.debug("Job dispatcher started...")
```

**Review by trgiangdo — DISMISSED** [review]
The PR looks good to me. I like the placement of the logs.

I only have 1 small suggestion.

File: taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py (hunk 2 above)
Comment (ooooo-create) [reply taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py]: I think may be `Job dispatcher starting...`  is better?

Comment (ooooo-create) [reply taipy/core/_orchestrator/_dispatcher/_job_dispatcher.py]: Yeah, i think `Job dispatcher started...` is better. thanks! What confuse me is the `...`  and ` started` seems not suitable

**Comment by trgiangdo** [comment]
You're right. We probably can remove both the `...`

**Review by trgiangdo — APPROVED** [review]

**Review by toan-quach — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**